pkgs/development/python-modules/pytorch/bin.nix

```diff
@@ -39,6 +42,17 @@ in buildPythonPackage {
     numpy
     pyyaml
     requests
+    typing-extensions
+  ];
+
+  # PyTorch are broken: the dataclasses wheel is required, but ships with
+  # Python >= 3.7. Our dataclasses derivation is incompatible with >= 3.7.
```

Comment:
That's because dataclasses are an upstream package since 3.8 and python3Packages.dataclasses is a drop-in replacement that is only supported <=3.7. So not really a problem of "Our" derivation.

https://github.com/ericvsmith/dataclasses/blob/master/setup.py#L23

Reply:
Can we add a pythonAtLeast here?

Reply:
@mweinelt dataclasses is already in 3.7. It’s our problem in the sense that regular pip would resolve this by installing a 3.7 or 3.8 compatible version (which is an older version). This is why it is working for upstream.

@SuperSandro2000 why? This derivation is limited to 3.7 and 3.8 anyway. See the assertion.